spotinst_sdk2/models/hpc/aws/__init__.py

```diff
@@ -0,0 +1,150 @@
+import json
+
+none = "c6791c1d3bcab7d4179f64f0f4d50f6091bfe7fadaa790ba8e98fc8645e2b576"
+
+# region Strategy
+class Strategy:
+    """
+    # Arguments
+    draining_timeout: int
+    """
+    def __init__(
+            self,
+            draining_timeout:int=none,
+            ):
+        self.draining_timeout = draining_timeout
+        
+# endregion
+
+# region Capacity
+class Capacity:
+    """
+    # Arguments
+    minimum: int
+    maximum: int
+    target: int
+    unit: str
+    """
+    def __init__(self, minimum:int=none, maximum:int=none, target:int=none, unit:str=none):
+
+        self.minimum = minimum
+        self.maximum = maximum
+        self.target = target
+        self.unit = unit
+
+# endregion
+
+# region Instance Types
+class InstanceTypes:
+    """
+    # Arguments
+    ondemand: str
+    spot: list[str]
+    """
+    def __init__(
+            self,
+            onDemand:str=none,
+            spot:list[str]=none):
+
+        self.onDemand = onDemand
+        self.spot = spot
+
+# endregion
+
+# region Launch Specification
+class LaunchSpecification:
+    """
+    # Arguments
+    security_group_ids: list[str]
+    image_id: Image
+    monitoring: bool
+    key_pair: str
+    user_data: str
+    """
+    def __init__(
+            self,
+            security_group_ids:list[str]=none,
+            image_id:str=none,
+            monitoring:bool=none,
+            key_pair:str=none,
+            user_data:str=none):
+
+        self.security_group_ids = security_group_ids
+        self.monitoring = monitoring
+        self.image_id = image_id
+        self.key_pair = key_pair
+        self.user_data = user_data
+
+# endregion
+
+
+
+# region Compute
+class Compute:
+    """
+    # Arguments
+    launch_specification: LaunchSpecification
+    instance_types: InstanceTypes
+    product: str
+    subnet_ids: list[str]
+    """
+    def __init__(
+            self,
+            launch_specification:LaunchSpecification=none,
+            instance_types:InstanceTypes=none,
+            product:str=none,
```

Comment:
Remove "product". It is removed from the JSON.

Reply:
removed